src/styles/pages/about.module.scss

```
@@ -0,0 +1,62 @@
+@use '../vars.scss' as vars;
+
+.logo {
+  height: 115;
+}
+
+.container {
+  font-size: vars.$general-body;
+  margin: 0 auto;
+  max-width: 60rem;
+}
+
+.header {
+  align-items: center;
+  display: flex;
+
+  h1 {
+    font-size: vars.$general-heading-2;
+    margin-left: 1.25rem;
+  }
+}
+
+.body {
+  margin: auto;
+  max-width: auto;
+
+  @media only screen and (min-width: vars.$breakpoint-md) {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .socials {
+    padding: 2rem;
+
+    .description {
+      margin-bottom: 2rem;
+      margin-top: 0;
+    }
+
+    a {
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+      gap: 0.5rem;
+      margin-bottom: 0.7rem;
+
+      .theme {
+        fill: currentColor;
+      }
+    }
+  }
+
+  .discordpreview {
```

```suggestion
  .discordPreview {
```
nitpick: camelcase CSS classes